requirements-dev.txt

```diff
@@ -17,3 +17,4 @@ diffusers
 pytorchcv
 librosa
 soundfile
+opencv-python-headless
```

Comment:
Maybe we can pin the package version? So the package won't change by surprise

Reply:
I am ok to allow a similar problem to appear again before we decide to pin the package version.
Wdyt?